aat/src/aat/java/uk/gov/hmcts/ccd/definitionstore/tests/functional/ImportDefinitionTest.java

```diff
@@ -31,4 +31,61 @@ void shouldNotImportInvalidDefinition() {
             .post("/import");
     }
 
+    @Test
+    @DisplayName("Should Not import a definition with missing Permissions")
+    void shouldNotImportInvalidDefinitionMissingComplexAuthorization() {
+
+        Supplier<RequestSpecification> asUser = asAutoTestImporter();
+        asUser.get()
+            .given()
+            .multiPart(new File("src/resource/CCD_CNP_27_Missing_Complex_Authorization.xlsx"))
+            .expect()
+            .statusCode(500)
```

Comment:
This error code should not be 500. It should be 400.
If this is returning 500, it is a defect that must be fixed.

Reply:
Probably lets update the functional tests as part of the fix

Reply:
Requested Nitish to complete test as it should be but leave it disabled. When fix is done, we should also enable the test back.